### PR TITLE
Feat/ SER-2040 add support of fetching client data in validation route

### PR DESCRIPTION
**Pull request by amydoxym**
## Description

add support of fetching client data in validation route

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+trigger:
+  - main
```
Comment (victorjbassey) [inline azure-pipelines.yml:2]: The target branch for this PR is `master`. Are you going to rename the branch to `main`?

Comment (amydoxym) [reply azure-pipelines.yml:2]: Decided to keep master to avoid potential mismatch since it's a fork repo

Comment (victorjbassey) [reply azure-pipelines.yml:2]: Understood. Btw, `main` or any other branch can always be set to track changes from the upstream repo's `master`

**Review by victorjbassey — APPROVED** [review]

File: azure-pipelines.yml (hunk 1 above)
Comment (amydoxym) [reply azure-pipelines.yml:2]: Thanks for the info

**Pull request merged**